README.md

```diff
@@ -25,6 +25,23 @@ or visit our
 [Support and Help](https://www.ably.io/support)
 site to discuss privately.
 
+## Supported Platforms
+
+#### iOS
+
+iOS9 or newer
+
+#### Android
+
+SDKVersion 19 or newer.
+
+_This project uses Java8 language features and uses [Desugaring](https://developer.android.com/studio/write/java8-support#library-desugaring)
+to support lower versions of android (< SDKVersion 24)_
+
+If your project needs support for lower SDK Version, Android Gradle Plugin 4.0.0+ must be used.
+You might also need to upgrade [gradle distribution](https://developer.android.com/studio/releases/gradle-plugin#updating-plugin) accordingly.
+
+
```

Comment:
Please avoid multiple new lines. One is always enough. 😁

Reply:
Addressed: a23b04ef17039c4a68d05220ea76366e3e0e974e